### PR TITLE
Extract style rules

### PR DESCRIPTION
**Pull request by almsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 {
   "parser": "babel-eslint",
   "extends": [
-    "@jetbrains",
+    "@jetbrains/eslint-config",
```
Comment (Hypnosphi) [inline README.md]: Why the old way doesn't work anymore?

Comment (andrey-skl) [reply README.md]: We use old way at least in Hub, I'd like to find a way to keep it unchanged

Comment (almsh) [reply README.md]: As i understand `@jetbrains` is just a scope name, not package, so does this actually works? If yes I will just remove this commit from PR.

Comment (andrey-skl) [reply README.md]: @almsh yes it works, it is feature of eslint shareable config
https://eslint.org/docs/developer-guide/shareable-configs#npm-scoped-modules

Comment (Hypnosphi) [reply README.md]: eslint resolves `foo` as `eslint-plugin-foo` and `@foo` as `@foo/eslint-plugin`

Comment (almsh) [reply README.md]: Thanks for clarification:)

**Comment by Hypnosphi** [comment]
Please mention new configs in docs

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**